### DISCUSSION TITLE
Laravel Console Facade

### DISCUSSION THREAD

**Discussion by henzeb** [HassanZahirnia/laravel-package-ocean#40] [Package Suggestions]
A facade that gives you access to all the sweet input and output functions you have inside Console Commands anywhere else, and more! It fixes a lot of issues you may encounter with vanilla console functions, it eases working with sections, verbose and debug messages, handles input validation (using laravel's validation system).

https://github.com/henzeb/laravel-console-facade

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊,
I see you've really worked hard on the package and kept it compatible with latest Laravel versions so I decided to add the package! It will be live in a moment.
